src/RtsClient.js

```diff
@@ -12,4 +12,21 @@ export default class RtsClient {
             json: true,
         });
     }
+
+    trackReferral(referrer, user_id, user_email) {
+        return request({
+            url: this._url + '/register',
+            json: true,
+            body: {referrer, user_id, user_email},
+            method: 'POST',
+        });
```

Comment:
The team server looks each email up with the identity server to check that it's a match to the given `user_id`. Any `user_id`/`user_email` pair can only be "referred" once. The email has to be a team email so unless someone manages to genuinely sign people up from their uni, the referral won't count.